src/web/app/page.component.html

```diff
@@ -42,25 +42,67 @@
   </div>
 </nav>
 <div id="main-content" class="container main-content">
-  <div *ngIf="isUnsupportedBrowser">
-    You are currently using {{ browser }}. This web browser is not officially supported by TEAMMATES.
-    In case this web browser does not display the webpage correctly you may wish to view it in the following supported browsers:
-    <ol>
-      <li>Microsoft Internet Explorer {{ minimumVersions['IE'] }}+</li>
-      <li>Chrome {{ minimumVersions['Chrome'] }}+</li>
-      <li>Firefox {{ minimumVersions['Firefox'] }}+</li>
-      <li>Safari {{ minimumVersions['Safari'] }}+</li>
-    </ol>
-    <br>
+  <div *ngIf="isUnsupportedBrowser && !alertDismissed">
+    <ngb-alert type="info" (close)="closeAlert()">
+      <i class="fas fa-exclamation-circle"></i>&nbsp;You are currently using {{ browser }}. This web browser is not officially supported by TEAMMATES.
+      In case this web browser does not display the webpage correctly you may wish to view it in the following supported browsers:
+      <ol>
+        <li>Microsoft Internet Explorer {{ minimumVersions['IE'] }}+</li>
+        <li>Chrome {{ minimumVersions['Chrome'] }}+</li>
+        <li>Firefox {{ minimumVersions['Firefox'] }}+</li>
+        <li>Safari {{ minimumVersions['Safari'] }}+</li>
+      </ol>
+    </ngb-alert>
   </div>
-  <div>
-    <p *ngIf="!(isNetworkOnline$ | async)">You are not connected to internet.</p>
+  <div *ngIf="!(isNetworkOnline$ | async)">
+    <div class="alert alert-danger" role="alert">
+      <i class="fas fa-wifi"></i>&nbsp;You are not connected to the internet.
+    </div>
   </div>
   <div *ngIf="isCookieDisabled">
-    You are currently not enabling cookies in your browser. TEAMMATES requires cookies in order to work effectively.
+    <div class="alert alert-danger" role="alert">
+      <i class="fas fa-ban"></i>&nbsp;You are currently not enabling cookies in your browser. TEAMMATES requires cookies in order to work effectively.
+    </div>
   </div>
-  <div *ngIf="!isValidUser">
-    Invalid user!
+  <div *ngIf="!isValidUser && (isStudent || isInstructor || isAdmin)">
+    <div class="row">
+      <div class="alert alert-light col-sm-4 offset-sm-4">
+        <img src="assets/images/angry.png" style="float: left; height: 90px; margin: 0 10px 10px 0;">
+        <h5 style="padding: 30px 0 20px 0">You are not authorized to view this page.</h5>
+        <hr/>
```

Comment:
Let's not close self-closing tags like `hr` and `br`, similarly for other places.

Reply:
Okay updated!